packages/yourloops/lib/patient/patient.api.ts

```diff
@@ -64,6 +64,22 @@ export default class PatientApi {
     }
   }
 
+  static async getPatient(userId: string): Promise<Patient[]> {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why would a `getPatient()` call return multiple results, especially if we pass the `userId` as an argument? Even if the API call returns only one result in an array, I think it would make more sense for the `getPatient()` method to return a `Promise<Patient>` instead of a `Promise<Patient[]>`

Reply:
I discuss this with Nasser. Making this method return a single patient has complex impacts in the code. The goal for now is to integrate this route, and then we will be free to take the time to improve this.
However, it might be better to have this method named `getPatients` for now